dymos/transcriptions/analytic/analytic.py

```diff
@@ -276,9 +281,10 @@ def setup_timeseries_outputs(self, phase):
                                                            output_subset=options['subset'],
                                                            time_units=phase.time_options['units'])
 
-            phase.add_subsystem(name, subsys=timeseries_comp)
+            timeseries_group = TimeseriesOutputGroup(has_expr=has_expr, timeseries_output_comp=timeseries_comp)
+            phase.add_subsystem(name, subsys=timeseries_group)
```

Comment:
use argument `promotes_inputs=['*'], promotes_outputs=['*']` here so that things can be addressed as `timeseries.foo` rather than `timeseries.timeseries_comp.foo` elsewhere in the code.  That should make the contents of the timeseries group irrelevant to the user.

As long as you do this promotion for both the enclosed exec comp and the enclosed TimeseriesComp, connections will not have to change and any common inputs between the two components will be implicitly connected.